iroha_crypto/src/lib.rs

```diff
@@ -33,7 +33,7 @@ use ursa::{
 };
 
 /// Length of hash
-pub const HASH_LENGTH: usize = 32;
+pub const HASH_LENGTH: usize = 2_usize.pow(5);
```

Comment:
And for HASH_LENGTH this number is derived from the ursa defaults, so it is better to leave it like this

Reply:
done